core/src/main/java/io/camunda/migrator/HistoryMigrator.java

```diff
@@ -307,7 +326,81 @@ private void migrateDecisionDefinition(DecisionDefinition legacyDecisionDefiniti
     }
   }
 
-  public void migrateIncidents() {
+  public void migrateDecisionInstances() {
+    HistoryMigratorLogs.migratingDecisionInstances();
+    if (RETRY_SKIPPED.equals(mode)) {
+      dbClient.fetchAndHandleSkippedForType(HISTORY_DECISION_INSTANCE, idKeyDbModel -> {
+        HistoricDecisionInstance historicDecisionInstance = c7Client.getHistoricDecisionInstance(idKeyDbModel.id());
+        migrateDecisionInstance(historicDecisionInstance);
+      });
+    } else {
+      c7Client.fetchAndHandleHistoricDecisionInstances(this::migrateDecisionInstance,
+          dbClient.findLatestStartDateByType((HISTORY_DECISION_INSTANCE)));
+    }
+  }
+
+  private void migrateDecisionInstance(HistoricDecisionInstance legacyDecisionInstance) {
+    if (legacyDecisionInstance.getProcessDefinitionKey() != null) {
+      // only migrate decision instances that were triggered by process definitions
```

Comment:
❓ Do we want to set a logging in this case? As at the moment, it's just ignored.

Reply:
✍️ I think we should also add it to the limitation list in the read me.